corehq/apps/userreports/tests/test_report_aggregation.py

```diff
@@ -868,16 +854,36 @@ def test_aggregate_date(self):
                 }
             ],
             filters=None,
+            sort_expression=[
+                {
+                    "field": "month",
+                    "order": "DESC"
+                },
+            ],
         )
         view = self._create_view(report_config)
-        self.assertEqual(
-            view.export_table,
-            [['foo',
-              [['report_column_display_state', 'month', 'report_column_display_number'],
-               ['MA', self._relative_month(1, year_offset=-1), 6],
-               ['MA', self._relative_month(2, year_offset=-1), 3],
-               ['TN', self._relative_month(1, year_offset=-2), 1]]]]
-        )
+        if self._relative_month(365 + 28 * 3 + 4) == self._relative_month(365 + 28 * 3 + 5):
```

Comment:
this would now run different tests based on this condition, yes?
Should we write the tests in a way that we can cover both cases?

also can we pull out dates meant to be consecutive as variables so that the intention with them is clear?

Reply:
Added constants in 2cc6885b33c and some more comments.

I agree this test could be better but I don't think it's worth spending more time on it. It'll now be a less useful test one day a month, whereas before it would entirely fail (incorrectly) on certain dates.